application/console/controllers/CronController.php

```diff
@@ -0,0 +1,43 @@
+<?php
+namespace console\controllers;
+
+use common\models\EmailQueue;
+use yii\console\Controller;
+
+class CronController extends Controller
+{
+    /**
+     * Retry sending emails from queue in small batch sizes
+     */
+    public function actionSendQueuedEmail()
+    {
+        try {
+            echo 'starting cron/send-queued-email' . PHP_EOL;
+
+            $batchSize = \Yii::$app->params['emailQueueBatchSize'];
```

Comment:
It would error, but currently it is hard coded in the main config so it will always be present.